packages/suite/src/actions/suite/initAction.ts

```diff
@@ -28,7 +27,7 @@ export const init = () => async (dispatch: Dispatch, getState: GetState) => {
     // right after storage is loaded, we might start:
 
     // 1. init analytics
-    dispatch(analyticsActions.init(analytics));
```

Comment:
is ok that you removed that param?

Reply:
Previously, state was passed through parameter and the thunk was picking properties from previous state. I decided it's not necessary when `getState` is available via thunk props. Consulted this with @tomasklim and he verified that there was probably no reason for this and could be refactored. It's better like this IMO.